data/Manazsharma.json

```
@@ -0,0 +1,21 @@
+{
+	"name": "Manas Sharma",
+	"type": "personal",
+	"bio": "Frontend Devloper| Open Source Enthusiast| Intrested in Web3 & Devops ",
+	"links": [{
+			"name": "Follow me on Twitter",
+			"url": "https://twitter.com/manassharma__",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Follow me on LindkedIn",
+			"url": "https://www.linkedin.com/in/manas-sharma-a49295220/",
+			"icon": "FaLinkedIn"
+		},
+		{
+			"name": "GitHub",
+			"url": "https://github.com/Manazsharma",
+			"icon": "FaGitHub"
```

Also, this should be `FaGithub`.

```suggestion
			"icon": "FaGithub"
```